sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -22,6 +22,8 @@ public partial class AnalyzeSentimentResultCollection : System.Collections.Objec
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
         public Azure.AI.TextAnalytics.SentimentConfidenceScores ConfidenceScores { get { throw null; } }
+        public int Length { get { throw null; } }
+        public int Offset { get { throw null; } }
```

Comment:
How does `Length` relate to `Text.Length`?

It'd be worth discussing in ApiView if there's a prefix we can use for `Offset` to make it clearer to new users of the service.  I also think .NET more commonly calls these things `Index` (like https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.capture.index?view=netcore-3.1#System_Text_RegularExpressions_Capture_Index) but that might be kind of dated now that we have System.Index.

Reply:
>How does Length relate to Text.Length?

There is no difference. That is a great point :| 
I will add both comments into the API View